app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -37,6 +40,67 @@ export default function DocumentDownloadMenu(props) {
   const ability = useContextualAbility();
   const canDownloadJournalPdf = ability.can('download-journal-pdf', atbd);
 
+  const handlePdfDownloadClick = React.useCallback(() => {
+    const { id, version, alias } = atbd;
+    const pdfUrl = `${apiUrl}/atbds/${id}/versions/${version}/pdf`;
+    const pdfFileName = `${alias}-v${version}.pdf`;
+    let retryCount = 0;
+    let timeoutRef;
+
+    async function fetchPdf(url) {
+      const toast = createProcessToast('Downloading PDF, please wait...');
+      const user = await Auth.currentAuthenticatedUser();
+      if (!user) {
+        toast.error('Failed to download PDF! (Not authenticated)');
+        return;
+      }
+
+      const {
+        signInUserSession: { idToken, accessToken }
+      } = user;
+      const authToken = `Bearer ${idToken.jwtToken}`;
+      const xAccessToken = accessToken.jwtToken;
+
+      try {
+        const response = await fetch(url, {
+          method: 'GET',
+          headers: {
+            Authorization: authToken,
+            'X-ACCESS-TOKEN': xAccessToken
+          }
+        });
+
+        if (response.headers['content-type'] === 'application/json') {
```

Comment:
```suggestion
        if (response.headers.get('content-type') === 'application/json') {
```
`response.headers['content-type']` is undefined. Might be because of https://stackoverflow.com/a/48432628